build.sbt

```diff
@@ -112,7 +144,10 @@ lazy val plugin = project
     scalaVersion                            := Scala212,
     libraryDependencies += "org.scalatest" %%% "scalatest" % ScalaTestVersion % Test,
     Compile / resourceGenerators += generateVersionFile.taskValue,
-    scriptedLaunchOpts += version.apply { v => s"-Dproject.version=$v" }.value,
+    scriptedLaunchOpts := {
+      scriptedLaunchOpts.value ++
+        Seq("-Xmx1024M", s"-Dplugin.version=${version.value}", s"-Dproject.version=${version.value}")
```

Comment:
Is `-Dplugin.version=${version.value}` actually needed?

Reply:
@tototoshi beating me to the feedback nice! I'll double check that one

Reply:
Looks like it's not needed so I removed it!